### PR TITLE
Run CI on Windows

### PR DESCRIPTION
**Pull request by chrisseaton**
Downstream gems such as `shopify-cli` need to run on Windows.

### CODE REVIEW

File: test/cli/kit/resolver_test.rb [hunk 1]
```diff
@@ -36,8 +36,8 @@ def test_resolver_no_match
       end
 
       def test_resolver_suggest
-        reg.add(32, 'floo')
```
Comment (lugray) [inline test/cli/kit/resolver_test.rb:39]: Could you please revert the changes in this file? The previous did a better job of showing ranked proximity to the provided command.

File: test/cli/kit/executor_test.rb [hunk 2]
```diff
@@ -74,14 +74,18 @@ def test_siginfo_handling
       end
 
       def test_sigquit_handling
-        @exe.expects(:exit).with(CLI::Kit::EXIT_FAILURE_BUT_NOT_BUG)
-        out, err = capture_io do
-          exe.call(->(*) { Process.kill('QUIT', Process.pid) }, 'foo', [])
+        if Signal.list.key?('INFO')
```
Comment (lugray) [inline test/cli/kit/executor_test.rb:77]: I'm confused by `INFO` here - should this be `QUIT`?

File: test/cli/kit/config_test.rb [hunk 3]
```diff
@@ -90,7 +90,8 @@ def test_config_set
         assert_equal("[section]\nsome-key = ~/.test\nsome-other-key = ~/.test", File.read(@file))
 
         assert_equal('~/.test', @config.get('section', 'some-key'))
-        assert_equal("#{ENV["HOME"]}/.test", @config.get_path('section', 'some-key'))
+        assert_equal(File.expand_path("#{ENV["HOME"]}/.test"),
+          File.expand_path(@config.get_path('section', 'some-key')))
```
Comment (lugray) [inline test/cli/kit/config_test.rb:94]: The point of this test was to check that `get_path` expands the path.
```suggestion
        assert_equal(File.expand_path('.test', ENV['HOME']), @config.get_path('section', 'some-key'))
```

File: test/cli/kit/resolver_test.rb (hunk 1 above)
Comment (chrisseaton) [reply test/cli/kit/resolver_test.rb:39]: The problem I was solving is that the current two examples have the same ranking. On Windows they get returned in different orders. I guess the order is undefined when they have the same ranking.

**Review by lugray — APPROVED** [review]
